routers/user/home.go

```diff
@@ -54,7 +54,7 @@ func getDashboardContextUser(ctx *context.Context) *models.User {
 }
 
 // retrieveFeeds loads feeds for the specified user
-func retrieveFeeds(ctx *context.Context, user *models.User, includePrivate, isProfile bool, includeDeletedComments bool) {
+func retrieveFeeds(ctx *context.Context, user *models.User, includePrivate, isProfile, includeCollaborate bool, includeDeletedComments bool) {
```

Comment:
It might be time to add an options type for this function. If I see something like `retrieveFeeds(ctx, ctxUser, true, false, true, false)`, I'll have no idea what it means without further investigating.

We could also reuse the `GetFeedsOptions` type instead of introducing a new options type, since most of the current argument map directly to `GetFeedsOptions` fields.